doc/nrf/known_issues.rst

```diff
@@ -518,21 +516,109 @@ Closing sockets
   Moreover, closing a socket does not properly clean up all memory resources.
   If a socket is opened and closed multiple times, this  might starve the system.
 
+Multiprotocol Service Layer (MPSL)
+==================================
+
+.. rst-class:: v1-3-1 v1-3-0
+
+KRKNWK-6361: Antenna Diversity not supported on nRF52840
+  The Antenna Diversity feature is not supported on nRF52840 devices.
+
+The following known issues and limitations have been moved from :ref:`MPSL's changelog <nrfxlib:mpsl_changelog>` for the |NCS| v1.4.0 release, and can also affect some of the previous versions of |NCS|.
+
+.. rst-class:: v1-4-0
+
+DRGN-6362: Do not use the synthesized low frequency clock source
+  The synthesized low frequency clock source is neither tested nor intended for usage with MPSL.
+
+.. rst-class:: v1-4-0
+
+DRGN-14153: Radio Notification power performance penalty
+  The Radio Notification feature has a power performance penalty proportional to the notification distance.
+  This means an additional average current consumption of about 600 �A for the duration of the radio notification.
+
+.. rst-class:: v1-4-0
+
+Front-End Modules API only for 802.15.4
+  The FEM API is currently implemented in the 802.15.4 protocol only.
+
+SoftDevice Controller
```

Comment:
All SoftDevice Controller issues mentioned here are valid for all earlier releases (NCS 1.0.0 - present). 
From the "Bugfixes" sections in earlier releases, the following known issues may be added:
```
.. rst-class:: v1-1-0

Several issues for nRF53:
 - Poor performance when performing active scanning
 - The controller could assert when receiving extended advertising packets
 - The T_IFS could in certain conditions be off by 5 us.
 - The radio could stay in TX state longer than expected. 
   This issue can only occur when sending a packet on either LE 1M or LE 2M PHY after receiving or transmitting a packet on LE Coded PHY. 
   If this occurs while performing a Link Layer Control Procedure, the controller could end up retransmitting an ACKed packet, resulting in a disconnect.

.. rst-class:: v1-1-0 v1-0-0

A control packet could be sent twice even after the packet was ACKed.
This would only occur if the radio was forced off due to an unforeseen condition.

.. rst-class:: v1-1-0 v1-0-0

The application is unable to restart a connectable advertiser right after a high duty cycle advertiser times out.

   **Workaround:** Wait 500 ms before restarting a connectable advertiser.

.. rst-class:: v1-1-0 v1-0-0

The controller could assert when receiving a packet with a CRC error after performing a data length update on Coded PHY.

.. rst-class:: v1-1-0 v1-0-0

The controller will assert when setting secondary PHY to 0 when using HCI LE Set Extended Advertising Parameters and the advertising type is set to legacy advertising

.. rst-class:: v1-1-0 v1-0-0

HCI LE Set Extended Scan Enable returns `UNSUPPORTED_FEATURE` when duplicate filtering is enabled.

.. rst-class:: v1-1-0 v1-0-0

HCI LE Set Advertising Parameters returns `UNSUPPORTED_FEATURE` when `secondary_max_skip` is set to a non-zero value.

.. rst-class:: v1-0-0

:c_func:`hci_data_get()` may return "No data available" when there is data available. 
This issue will only occur when connected to multiple devices at the same time.

.. rst-class:: v1-0-0

The controller will assert if the host issues LE Write Suggested Default Data Length.

.. rst-class:: v1-0-0

The controller does not indicate support for HCI LE Set Privacy Mode although it is supported.

.. rst-class:: v1-0-0

The controller will assert if advertising data is set after HCI Reset without first setting advertising parameters.

.. rst-class:: v1-0-0

The controller will assert if advertising data is set after HCI Reset without first setting advertising parameters.

.. rst-class:: v1-0-0

The controller may assert when writing to flash.

.. rst-class:: v1-0-0

A directed advertiser may time out without sending a packet on air.


```

Reply:
Updated